src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String line = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final Scanner input = new Scanner(System.in);
+
+    private static final List<Task> tasks = new ArrayList<Task>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        welcomeMessage();
+
+        String userInput;
+         do {
+            System.out.println(line);
+            userInput = input.nextLine().trim();
+            System.out.println(line);
+        } while(userCommand(userInput));
+    }
+
+    /**
+     * Prints welcome message for the bot.
+     */
+    private static void welcomeMessage() {
```

Comment:
Instead of just welcomeMessage(), I think you can try naming it printWelcomeMessage() so that the methods are verbs and not nouns? :)

Reply:
Okie thanks for the feedback will look through the name of my methods.